forge/tests/forge/library/seq.frg

```diff
@@ -1,4 +1,10 @@
-#lang forge/bsl
+#lang forge
+
+option run_sterling off
+/bsl
```

Comment:
Uh oh

Reply:
Oops, bad regex on my end when making the change